python-package/Readme.rst

```diff
@@ -221,25 +223,26 @@ CMake
 
 ::
 
-    cd rgf/include/fast_rgf/build
+    cd rgf/python-package/include/fast_rgf
+    mkdir build
+    cd build
     cmake ..
     make 
     make install
 
 Docker image
 ^^^^^^^^^^^^
 
-We provide `docker image <https://github.com/RGF-team/rgf/blob/master/docker/Dockerfile>`__ with installed **rgf\_python**.
+We provide `docker image <https://github.com/RGF-team/rgf/blob/master/python-package/docker/Dockerfile>`__ with installed **rgf\_python**.
 
 ::
 
     # Run docker image
     docker run -it RGF-team/rgf /bin/bash
```

Comment:
I'm not familiar with docker. Should something here be modified?

Reply:
Dockerhub is not updated since we moved repository, and `docker run -it RGF-team/rgf /bin/bash` doesn't work. I'm investigating about it.

Reply:
So, could we merge this PR, since it's another issue?

Reply:
It should be `docker run -it fukatani/rgf_python /bin/bash`.
https://hub.docker.com/r/fukatani/rgf_python/

I will make fukatani/rgf in near future.

Reply:
Couldn't it be `RGF-team/rgf_python`? I mean, create new account.